src/status_im/chat/handlers/commands.cljs

```diff
@@ -38,11 +40,14 @@
                 params   {:parameters params
                           :context    (generate-context db chat-id to)}
                 callback #(let [result (get-in % [:result :returned])
-                                result (if (:markup result)
+                                result' (if (:markup result)
                                          (update result :markup cu/generate-hiccup)
-                                         result)]
-                            (dispatch [:set-in [:message-data data-type message-id] result])
-                            (when on-requested (on-requested result)))]
+                                         result)] 
+                            (dispatch [:set-in [:message-data data-type message-id] result'])
+                            (when (= :preview data-type)
```

Comment:
tbh this `on-requested` name doesn't make too much sense for me at all, and it's a bit not clear how it is used in project. Like we will not solve the problem if we just move `messages/update` to another handler.  I don't mind if we merge this PR in it's current state (let's just fix these long lines) if we are going to refactor these handlers in the nearest future.